src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -99,12 +101,29 @@ public WindowImpl()
             };
             _rendererLock = new ManagedDeferredRendererLock();
 
+            _isUsingComposition = Win32Platform.WindowsVersion.Major >= 10 && Win32Platform.WindowsVersion.Build >= 14393 &&
```

Comment:
CreateDispatcherQueueController was introduced only in 16299 (1709)

```suggestion
            _isUsingComposition = Win32Platform.WindowsVersion.Major >= 10 && Win32Platform.WindowsVersion.Build >= 16299 &&
```